composables/useDoAfterlogin.ts

```diff
@@ -1,11 +1,16 @@
 import { openConnectWalletModal } from '@/components/common/ConnectWallet/useConnectWallet'
 
 export default function (instance) {
-  const doAfterLogin = (callback: (account?: string) => void) => {
+  const doAfterLogin = (
+    callback: (account?: string) => void,
+    onCancel?: () => void
+  ) => {
     const { isLogIn } = useAuth()
     if (!isLogIn.value) {
       openConnectWalletModal(instance, {
         onConnect: callback,
+        onCancel,
+        onClose: onCancel,
```

Comment:
still akward.
`openConnectWalletModal` doesn't need both `onCancel` and `onClose`

Reply:
Not resolved yet

Reply:
check now

Reply:
missed that check now